server/controllers/profile.controllers.js

```diff
@@ -0,0 +1,130 @@
+const Profile = require("../models/profile.model");
+
+module.exports = {
```

Comment:
I would recommend that you define the functions outside of this and then export them at the end

eg. 

```
const createProfile = async (req, res) => {...}

const retrieveProfile: async (req, res) => {...}

exports.createProfile = createProfile;
exports.retrieveProfile = retrieveProfile

```
This is more readable in my opinion, and is consistent with the way @Vaibhav-Kambli has already set it up

Reply:
I will refactor it. Thank you. 👍